README.md

```diff
@@ -25,14 +25,15 @@ of services are used. The main objective here is to simulate web applications us
 * **prefix_results_name**: A pattern to simulator use it as a prefix in result names.
 * **data_path**: The path where the CSV file of the experimental results is kept.
 * **input_file_name**: The name of the CSV file with the service time and status http of each request processed in an experiment.
+* **number_of_input_files**: The number of CSV files that exists to be used.  
```

Comment:
Much better to specify the list of file paths. We should rely on file naming standards unnecessarily.

Could you please change this parameter? I believe it is not hard.

Reply:
Thanks again! I trully agree with you. That's a better approach since the simulator does not need define a pattern of input.